docs/tutorials/otio-serialized-schema.md

```diff
@@ -354,6 +354,29 @@ parameters:
 - *name*: 
 - *parameters*: 
 
+### ImageSequenceReference.1
+
+*full module path*: `opentimelineio.schema.ImageSequenceReference`
+
+*documentation*:
+
+```
+None
+```
+
+parameters:
```

Comment:
can we get some of the documentation to flow through the python bindings?

Reply:
Added docstrings in the bindings. I'd love to have someone give feedback on wether or not it all makes sense. The bulk of the content is forklifted from this PR.